x-pack/plugins/apm/server/lib/correlations/queries/field_stats/get_field_stats.test.ts

```diff
@@ -20,7 +20,7 @@ const params = {
   includeFrozen: false,
   environment: ENVIRONMENT_ALL.value,
   kuery: '',
-  samplerShardSize: 5000,
+  samplerShardSize: 10000,
```

Comment:
Can we use `SAMPLER_SHARD_SIZE` from `x-pack/plugins/apm/common/correlations/constants.ts`?